cmd/manager/kodata/knative-serving/0.12.1.yaml

```diff
@@ -1864,46 +1864,6 @@ rules:
   - patch
   - watch
 
----
-apiVersion: networking.istio.io/v1alpha3
-kind: Gateway
-metadata:
-  labels:
-    networking.knative.dev/ingress-provider: istio
```

Comment:
Can we remove everything that has this label from the manifest?

Reply:
Sure, I have removed ClusterRole and Configmap for networking-istio, that have the label.